File: src/main/solved/Programmers/level2/javascript/124 나라의 숫자.js
function solution(n) {
    var answer = '';
    let store = ['4', '1', '2'];

    while(n != 0){
        answer = store[n%3] + answer;
        if(n%3==0) n-=3;
        n = parseInt(n/3);
    }
    return answer;
}

File: src/main/solved/Programmers/level2/java/124 나라의 숫자.java
class Solution {
    public String solution(int n) {
        String[] store = {"4", "1", "2"};
        String answer="";

        while(n > 3) {
            int rest = n%3;
            answer = store[rest] + answer;
            if(rest == 0)
                n-=3;
            n /= 3;
        }
        answer = store[n%3]+ answer;
        return answer;
    }
}